<launch>
    <node name="erp_base" pkg="erp_driver" type="ErpSerialHandler.py" output="screen">
        <param name="/port" value="/dev/ttyUSB0" type="str"/>
        <param name="/baudrate" value="115200" type="int" />
    </node>
</launch>
